helm/korifi/values.yaml

```diff
@@ -44,6 +44,16 @@ api:
       idle: 900
       readHeader: 10
 
+  infoConfig:
+    build: ""
```

Comment:
The API docs are not really clear on what `build` semantic is. Is there a use-case where this value is used? If not, we would prefer a lean approach and only put stuff that are actually needed.

Other than that, the docs example value sounds like a commit SHA. If that is the case, the same approach as to the `version` should be applied (i.e. should not be configurable via a helm value)

Reply:
I think the v3/info endpoint in general is very open for interpretation and how you as an operator use and configure it. I think the 'old' approach of cf-deployment to put here the current release number is nice:

https://github.com/cloudfoundry/cf-deployment/commit/ab937830c6f4d66d55ae9bc6d2d270255ab53d5c

For my future usage, I would probably put here not a Korifi specific hash tag alone, but my own interpretation of the current build (some semver for my current Korifi + Kubernetes + cert-manager + ... setup).